lib/lcache.ts

```diff
@@ -66,5 +66,5 @@ const cache: FastifyPluginCallback<ICacheOptions> = (
  */
 export default fp(cache, {
   name: '@fastify/lcache',
-  fastify: '3.x',
+  fastify: '4.x',
```

Comment:
I'm not sure but we can check if this plugin works on fastify v3 with changes below and specify `>=3.x` here. Have to check backwards compitability. Otherwise leave `4.x` as it is. What do you think @denbon05 ?